docs/client-libraries-java.md

```diff
@@ -4,1316 +4,37 @@ title: Pulsar Java client
 sidebar_label: "Java"
 ---
 
-````mdx-code-block
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-````
+You can use a Pulsar Java client to create Pulsar producers, consumers, and readers in Java and perform [administrative tasks](admin-api-overview.md). All the methods in Java clients are thread-safe. The current Java client version is **@pulsar:version@**.
```

Comment:
Let's add hyperlinks to what is Producer, Consumer and Reader (should be in concept pages).

Reply:
Added and applied to other languages.